docs/fundamentals/dashboard/enable-browser-telemetry.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Enable browser telemetry
+description: Learn how to enable browser telemetry in the .NET Aspire dashboard.
+ms.date: 10/03/2024
+---
+
+# Enable browser telemetry
```

Comment:
Changes here were too large for a review. I have a PR: https://github.com/dotnet/docs-aspire/pull/1808. Cherry-pick my commit(s)